src/om_css/core.cljc

```diff
@@ -28,9 +23,17 @@
 (defn local-kw
   "Generate a keyword for a localized CSS class for use in Garden CSS syntax as a localized component classname keyword."
   ([comp-class]
-   (keyword (str "." (cssify (fq-component comp-class)))))
+   (str "." (cssify (fq-component comp-class))))
```

Comment:
So, it is not acceptable to change a public-facing API function to do something different than it used to do. It is OK to augment it. It is OK to add a new function. But, this function is supposed to generate a keyword, and it must continue to do so.